doc/sphinx/introduction.rst

```diff
@@ -120,136 +120,193 @@ contained in the :ref:`Developers guide`.
 
-::
+.. rubric:: espressomd.System
 
-    system.box_l =(10.0,10.0,15.0) print system.time_step
+Access to the simulation system is provided via the System class. As a
+first step, an instance of this class needs to be created. ::
 
-The particles in the simulation are accessed via the ParticleList class.
-It is used to retrieve individual particles of the simulation as well as
-for adding particles. An instance of the class is provided as the part
-attribute of the System class. Individual particles can be retrieved by
-their numerical id by using angular brackets
+    system = espressomd.System(box_l = [10,10,10])
 
-::
+Note that only one instance of the System class can be created due to
+limitations in the simulation core. :ref:`Properties of the System
+class<Setting global variables in Python>` are used to access the parameters
+concerning the simulation system such as box geometry, time step or :ref:`cell-system<Cellsystems>`: ::
 
-    p=system.part[0]
+    print("The box dimensions are {}".format(system.box_l))
+    system.time_step = 0.01
+    system.cellsystem.skin = 0.4
 
-It is also possible to loop over all particles
+.. rubric:: Particles
 
-::
+The particles in the simulation are accessed via the ParticleList class. Use
+the `add` method to :ref:`create new particles<Adding particles>`: ::
 
-    for p in system.part: ...
+    system.part.add(id = 0, pos = [1.0, 1.0, 1.0], type = 0) 
+    system.part.add(id = 1, pos = [1.0, 1.0, 2.0], type = 0) 
 
-Particles are added via the add method
+Individual particles can be retrieved by their numerical id using angular
+brackets::
+    
+    system.part[1].pos = [1.0, 1.0, 2.0]
 
-::
+It is also possible to :ref:`loop<Iterating over particles and pairs of
+particles>` over all particles::
 
-    p=system.part.add(id=1,pos=(3.0,0.5,1.0),q=1)
+    for p in system.part:
+        print("Particle id {}, type {}".format(p.id, p.type))
 
 An individual particle is represented by an instance of ParticleHandle.
-The properties of the particle are implemented as Python properties:
+The properties of the particle (see
+:class:`espressomd.particle_data.ParticleHandle`) are implemented as Python
+properties. ::
 
-::
+    particle = system.part[0]
+    particle.type = 0
+    print("Position of particle 0: {}".format(particle.pos))
 
-    p=system.part[0] p.pos=(0,0,0) print p.id,p.pos system.part[0].q=-1
+:ref:`Properties of several particles<Interacting with groups of particles>`
+can be accessed by using Python slices: ::
 
-Properties of several particles can be accessed by using Python ranges
+    positions = system.part[:].pos
 
-::
+.. rubric:: Interactions
 
-    v=system.part[:].v
+Interactions between particles usually fall in three categories:
 
-Interactions between particles fall in three categories:
-
--  Non-bonded interactions are short-ranged interactions between *all*
+-  :ref:`Non-bonded interactions` are short-ranged interactions between *all*
```

Comment:
From a logical point of view non-bonded interactions contain short and long ranged interactions.